.github/workflows/integrationChartBump.yaml

```diff
@@ -0,0 +1,75 @@
+name: Integration Chart Promotion
+env:
+  charts: gcloud-sqlproxy datarepo-api datarepo-ui oidc-proxy
+
+on:
+  workflow_dispatch: {}
+
+jobs:
+  indivdual_chart_promotion:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: 'Checkout jade-data-repo repo'
+        uses: actions/checkout@v2
+        with:
+          repository: 'databiosphere/jade-data-repo'
+          token: ${{ secrets.CR_TOKEN }}
+          path: jade-data-repo
+          persist-credentials: false
+
+      - name: "Find latest version chart version"
+        id: chartversion
+        run: |
+          for i in $(echo ${{ env.charts }})
+          do
+            echo "::set-output name=$i-version::$(docker run --rm -v ${PWD}:/workdir mikefarah/yq yq read charts/$i/Chart.yaml version)"
+          done
+
+      - name: "Find and replace datarepo-api chartVersion version in integration action"
+        uses: docker://mikefarah/yq:latest
+        with:
+          args: yq w -i jade-data-repo/.github/workflows/int-and-connected-test-run.yml jobs.deploy_test_integration.steps[4].with.helm_datarepo_api_chart_version ${{ steps.chartversion.outputs.datarepo-api-version }}
```

Comment:
This seems like it might be brittle.  If we add a step in int-and-connected-test-run.yml, this would fail silently no?

Reply:
yeah i was lazy, if your yq foo is better please help.

Reply:
oh, it's definitely not better :) Lemme take a quick look

Reply:
Agreed - Maybe we could add an id for that step? The more robust we can make actions, the better.

Reply:
This seems to work:
```yq w -i jade-data-repo/.github/workflows/int-and-connected-test-run.yml 'jobs.deploy_test_integration.steps.(name==Deploy to cluster with Helm).with.helm_datarepo_api_chart_version' ${{ steps.chartversion.outputs.datarepo-api-version }}```